test/comptime.civet

```diff
@@ -0,0 +1,141 @@
+{ testCase, throws } from ./helper.civet
+{ compile } from ../source/main.civet
+{ serialize } from ../source/parser/comptime.civet
+assert from assert
+
+declare global
+  var outside: number
+
+describe "comptime", ->
+  options := parseOptions: comptime: true
+
+  testCase """
+    statement with indented block
+    ---
+    comptime
+      x := 5
+      x * x
+    ---
+
+  """, options
+
+  testCase """
+    one-line statement
+    ---
+    comptime x := 5
+    ---
+
+  """, options
+
+  it "statement has side effect", =>
```

Comment:
Eventually we wouldn't want `comptime` to have side effects within the same node process but we can handle that later.

Reply:
Perhaps. I was thinking of switching to `import('javascript:...')` (I feel like I saw this somewhere recently but not sure where?) instead of `eval` so it'd at least be in a separate module. Here I just want to measure that the code actually executes. If it ends up being out of process, we could write a file or something, but we'll cross that bridge when it comes.